mopp/modules/align.py

```diff
@@ -11,12 +11,6 @@
 
 def align_files(indir, outdir, pattern, INDEX, nthreads, compress=True):
     outdir_aligned = Path(outdir)
-    create_folder(outdir_aligned)
```

Comment:
thanks for identifying the issue! I think a simpler fix would be import `create_folder_without_clear` from `mopp.modules.utils` and then substitute all `create_folder` with `create_folder_without_clear`